.github/workflows/build.yml

```
@@ -91,9 +92,35 @@ jobs:
                     restore-keys: |
                         ${{ runner.os }}-php-${{ matrix.php }}-composer-
 
+            -
+                name: Prepare migrations for Sylius 1.7
+                if: matrix.sylius == '1.7.*'
+                run: |
+                    cp src/Migrations/* tests/Application/src/Migrations/
+                    cd tests/Application/src/Migrations
+                    sed -i -- 's/Sylius\\RefundPlugin\\Migrations/DoctrineMigrations/g' *
+                    cd ../../../../
```

Can be simplified with :
```suggestion
                    sed -i -- 's/Sylius\\RefundPlugin\\Migrations/DoctrineMigrations/g' tests/Application/src/Migrations/*
```